### PR TITLE
Polish commits!

### PR DESCRIPTION
**Pull request by Priultimus**
# Polish commits!

### Documentation changes
Essentially polished the README to make it more attractive, and added a license. 
* Added two buttons to the README
* Added easier to follow development instructions.

### Framework changes
All the code was run through black, added some support for a new help command and removed pathing.
* Everything preexisting was updated to use black as formatting.
* `handlers/paginator.py` and `handlers/helpformatter.py` were added to assist the new help command.
* Consistent error messages are now used across the bot, courtesy of @Critiql

### Code changes
In vein of all the other updates, polished everything and ran it through black to make all the commands cleaner. 
* Added a new help command, this one works and functions a lot better than the previous one. 
* Added another contact channel case.
* Changed the tick and x emotes.
* Made the prefix commands actually functional and require `MANAGE_MESSAGES` to use, courtesy of @inexaura

### Other changes
Established our new formatting system, and officially became open source.
* We added the GNU AFFERO GENERAL PUBLIC LICENSE, making Flux open source. 
* Introduced `.flake8` for our personal formatting expectations.
* Changed `.travis.yml` to run `black --check` as well as `flake8 .` to check the code.

Overall, we've switched to black (thanks @erebid), became open source because of @Xithrius & @erebid, and had some outside collaborative help from @inexaura and @Critiql.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,40 @@
...
-
-### Rules
-
-- Don't copy any code at all, it should all be written by you.
-- Only use open source libraries.
-- Make sure to keep up with the task assigned to you.
+```bash
+git clone https://github.com/tempus-dev/flux-discordbot
+cd flux-discordbot
+python3 -m pip install -r requirements.txt
```
Comment (Antonio32A) [inline README.md:36]: Use a python-venv instead. This isn't required but it is very recommended

File: handlers/paginator.py [hunk 2]
```diff
@@ -0,0 +1,330 @@
+import asyncio
```
Comment (Antonio32A) [inline handlers/paginator.py:1]: This is nice, but discord.ext.menus exists which may be a better option instead of this.

File: README.md (hunk 1 above)
Comment (Priultimus) [reply README.md:36]: I'll get around to that when I figure out docker for the bot.

File: handlers/paginator.py (hunk 2 above)
Comment (Priultimus) [reply handlers/paginator.py:1]: That didn't exist when the code was written hahaha

**Pull request merged**